software/nginx/en.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "nginx"
+lastmod = "2020-04-03T23:40:51+02:00"
++++
+# nginx
+Solus provides the nginx web server and proxy to users and developers who want to test their web applications before setting them to production. \
+Remember that Solus is not a server Operating System: any usage outside the local testing is not encouraged.
+
+## Installation
+nginx is available in the `nginx` package. It can be installed from the [Software Center](https://getsol.us/articles/common-tasks/installing-and-updating-software/en/) or via terminal:
+
+```bash
+sudo eopkg install nginx
+```
+
+## Usage
+
+### Configuration
+nginx, as many other packages in the Solus repository, is [stateless](https://clearlinux.org/features/stateless). That means the package gets installed with a vendored configuration out-of-the-box which can be customized, or even overridden, by a privileged user. \
+nginx on Solus is designed to read users' configuration files from `/etc/nginx/conf.d/` and `/etc/nginx/sites-enabled`. It is possible to manually create these directories and put personal files in there to customize nginx's behavior. To revert back to the original settings, just delete `/etc/nginx/`.
+
+### Management
+Basic nginx management is done via systemd, using the following commands:
+
+```bash
+sudo systemctl enable nginx # To enable nginx on startup.
+sudo systemctl start nginx  # To start nginx immediately.
```

Comment:
Can just be `sudo systemctl enable --now nginx`

Reply:
I was considering the snipped as a collection of commands rather than a batch script. OK! I will change chat.